api/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your models here.
+ACTIVITY_LEVEL = (
+			("SC", "Sleepy_Cat"),
+			("LC", "Lazy_Cat"),
+			("AC", "Active_Cat"),
+			("MC", "Mentor_Cat"),
+	)
+
+class UserProfile(models.Model):
+	user = models.OneToOneField(User) 
```

Comment:
Do you need to create the primary key in the table? I believe Django does that in the background. 
>primary_key¶

Field.primary_key¶
If True, this field is the primary key for the model.

If you don’t specify primary_key=True for any field in your model, Django will automatically add an AutoField to hold the primary key, so you don’t need to set primary_key=True on any of your fields unless you want to override the default primary-key behavior. For more, see Automatic primary key fields.

primary_key=True implies null=False and unique=True. Only one primary key is allowed on an object.

The primary key field is read-only. If you change the value of the primary key on an existing object and then save it, a new object will be created alongside the old one.